[2025-06-01T17:01:20.622+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-01T17:01:21.345+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: log_monitoring_dag.start_spark_consumer scheduled__2025-06-01T16:50:00+00:00 [queued]>
[2025-06-01T17:01:21.610+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: log_monitoring_dag.start_spark_consumer scheduled__2025-06-01T16:50:00+00:00 [queued]>
[2025-06-01T17:01:21.611+0000] {taskinstance.py:2865} INFO - Starting attempt 2 of 2
[2025-06-01T17:01:45.090+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): start_spark_consumer> on 2025-06-01 16:50:00+00:00
[2025-06-01T17:01:45.487+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=340) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-06-01T17:01:45.936+0000] {standard_task_runner.py:72} INFO - Started process 346 to run task
[2025-06-01T17:01:46.459+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'log_monitoring_dag', 'start_spark_consumer', 'scheduled__2025-06-01T16:50:00+00:00', '--job-id', '47', '--raw', '--subdir', 'DAGS_FOLDER/log_monitoring_dag.py', '--cfg-path', '/tmp/tmpag922f0r']
[2025-06-01T17:01:46.509+0000] {standard_task_runner.py:105} INFO - Job 47: Subtask start_spark_consumer
[2025-06-01T17:01:46.613+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-06-01T17:01:47.189+0000] {task_command.py:467} INFO - Running <TaskInstance: log_monitoring_dag.start_spark_consumer scheduled__2025-06-01T16:50:00+00:00 [running]> on host 61a0313acf02
[2025-06-01T17:01:47.288+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-06-01T17:01:51.509+0000] {job.py:229} INFO - Heartbeat recovered after 46.50 seconds
[2025-06-01T17:02:00.224+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='log_monitoring_dag' AIRFLOW_CTX_TASK_ID='start_spark_consumer' AIRFLOW_CTX_EXECUTION_DATE='2025-06-01T16:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-01T16:50:00+00:00'
[2025-06-01T17:02:00.225+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-01T17:04:39.661+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-06-01T17:04:39.718+0000] {job.py:229} INFO - Heartbeat recovered after 168.37 seconds
[2025-06-01T17:04:39.764+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'docker run --rm --network realtime_log_monitoring_default -v /opt/***/app:/app bitnami/spark:3.3.0 spark-submit --packages org.apache.spark:spark-sql-kafka-0-10_2.12:3.2.0 /app/log_consumer.py']
[2025-06-01T17:04:39.917+0000] {subprocess.py:86} INFO - Output:
[2025-06-01T17:04:44.340+0000] {job.py:229} INFO - Heartbeat recovered after 167.60 seconds
[2025-06-01T17:05:48.239+0000] {job.py:229} INFO - Heartbeat recovered after 51.42 seconds
[2025-06-01T17:05:49.376+0000] {subprocess.py:93} INFO - [38;5;6m [38;5;5m17:05:49.34 [0m
[2025-06-01T17:05:49.377+0000] {subprocess.py:93} INFO - [38;5;6m [38;5;5m17:05:49.37 [0m[1mWelcome to the Bitnami spark container[0m
[2025-06-01T17:05:49.391+0000] {subprocess.py:93} INFO - [38;5;6m [38;5;5m17:05:49.37 [0mSubscribe to project updates by watching [1mhttps://github.com/bitnami/containers[0m
[2025-06-01T17:05:49.393+0000] {subprocess.py:93} INFO - [38;5;6m [38;5;5m17:05:49.39 [0mSubmit issues and feature requests at [1mhttps://github.com/bitnami/containers/issues[0m
[2025-06-01T17:05:49.395+0000] {subprocess.py:93} INFO - [38;5;6m [38;5;5m17:05:49.39 [0m
[2025-06-01T17:05:49.406+0000] {subprocess.py:93} INFO - 
[2025-06-01T17:05:53.354+0000] {job.py:229} INFO - Heartbeat recovered after 50.46 seconds
[2025-06-01T17:06:07.790+0000] {subprocess.py:93} INFO - :: loading settings :: url = jar:file:/opt/bitnami/spark/jars/ivy-2.5.0.jar!/org/apache/ivy/core/settings/ivysettings.xml
[2025-06-01T17:06:07.862+0000] {subprocess.py:93} INFO - Ivy Default Cache set to: /opt/bitnami/spark/.ivy2/cache
[2025-06-01T17:06:07.863+0000] {subprocess.py:93} INFO - The jars for the packages stored in: /opt/bitnami/spark/.ivy2/jars
[2025-06-01T17:06:07.868+0000] {subprocess.py:93} INFO - org.apache.spark#spark-sql-kafka-0-10_2.12 added as a dependency
[2025-06-01T17:06:07.869+0000] {subprocess.py:93} INFO - :: resolving dependencies :: org.apache.spark#spark-submit-parent-8ae41390-c417-42fa-a2c7-9965c4dd3135;1.0
[2025-06-01T17:06:07.870+0000] {subprocess.py:93} INFO - 	confs: [default]
[2025-06-01T17:06:26.047+0000] {job.py:229} INFO - Heartbeat recovered after 18.77 seconds
[2025-06-01T17:06:28.417+0000] {subprocess.py:93} INFO - 	found org.apache.spark#spark-sql-kafka-0-10_2.12;3.2.0 in central
[2025-06-01T17:06:30.937+0000] {subprocess.py:93} INFO - 	found org.apache.spark#spark-token-provider-kafka-0-10_2.12;3.2.0 in central
[2025-06-01T17:06:35.755+0000] {subprocess.py:93} INFO - 	found org.apache.kafka#kafka-clients;2.8.0 in central
[2025-06-01T17:06:39.556+0000] {subprocess.py:93} INFO - 	found org.lz4#lz4-java;1.7.1 in central
[2025-06-01T17:06:39.854+0000] {subprocess.py:93} INFO - 	found org.xerial.snappy#snappy-java;1.1.8.4 in central
[2025-06-01T17:06:41.387+0000] {subprocess.py:93} INFO - 	found org.slf4j#slf4j-api;1.7.30 in central
[2025-06-01T17:06:44.688+0000] {subprocess.py:93} INFO - 	found org.apache.hadoop#hadoop-client-runtime;3.3.1 in central
[2025-06-01T17:06:46.212+0000] {subprocess.py:93} INFO - 	found org.spark-project.spark#unused;1.0.0 in central
[2025-06-01T17:06:46.524+0000] {subprocess.py:93} INFO - 	found org.apache.hadoop#hadoop-client-api;3.3.1 in central
[2025-06-01T17:06:49.171+0000] {subprocess.py:93} INFO - 	found org.apache.htrace#htrace-core4;4.1.0-incubating in central
[2025-06-01T17:06:51.941+0000] {subprocess.py:93} INFO - 	found commons-logging#commons-logging;1.1.3 in central
[2025-06-01T17:06:52.226+0000] {subprocess.py:93} INFO - 	found com.google.code.findbugs#jsr305;3.0.0 in central
[2025-06-01T17:06:54.937+0000] {subprocess.py:93} INFO - 	found org.apache.commons#commons-pool2;2.6.2 in central
[2025-06-01T17:06:55.084+0000] {subprocess.py:93} INFO - downloading https://repo1.maven.org/maven2/org/apache/spark/spark-sql-kafka-0-10_2.12/3.2.0/spark-sql-kafka-0-10_2.12-3.2.0.jar ...
[2025-06-01T17:06:55.592+0000] {subprocess.py:93} INFO - 	[SUCCESSFUL ] org.apache.spark#spark-sql-kafka-0-10_2.12;3.2.0!spark-sql-kafka-0-10_2.12.jar (642ms)
[2025-06-01T17:06:55.729+0000] {subprocess.py:93} INFO - downloading https://repo1.maven.org/maven2/org/apache/spark/spark-token-provider-kafka-0-10_2.12/3.2.0/spark-token-provider-kafka-0-10_2.12-3.2.0.jar ...
[2025-06-01T17:06:55.881+0000] {subprocess.py:93} INFO - 	[SUCCESSFUL ] org.apache.spark#spark-token-provider-kafka-0-10_2.12;3.2.0!spark-token-provider-kafka-0-10_2.12.jar (287ms)
[2025-06-01T17:06:56.018+0000] {subprocess.py:93} INFO - downloading https://repo1.maven.org/maven2/org/apache/kafka/kafka-clients/2.8.0/kafka-clients-2.8.0.jar ...
[2025-06-01T17:06:58.075+0000] {subprocess.py:93} INFO - 	[SUCCESSFUL ] org.apache.kafka#kafka-clients;2.8.0!kafka-clients.jar (2193ms)
[2025-06-01T17:06:58.210+0000] {subprocess.py:93} INFO - downloading https://repo1.maven.org/maven2/com/google/code/findbugs/jsr305/3.0.0/jsr305-3.0.0.jar ...
[2025-06-01T17:06:58.358+0000] {subprocess.py:93} INFO - 	[SUCCESSFUL ] com.google.code.findbugs#jsr305;3.0.0!jsr305.jar (283ms)
[2025-06-01T17:06:58.496+0000] {subprocess.py:93} INFO - downloading https://repo1.maven.org/maven2/org/apache/commons/commons-pool2/2.6.2/commons-pool2-2.6.2.jar ...
[2025-06-01T17:06:58.677+0000] {subprocess.py:93} INFO - 	[SUCCESSFUL ] org.apache.commons#commons-pool2;2.6.2!commons-pool2.jar (317ms)
[2025-06-01T17:06:58.822+0000] {subprocess.py:93} INFO - downloading https://repo1.maven.org/maven2/org/spark-project/spark/unused/1.0.0/unused-1.0.0.jar ...
[2025-06-01T17:06:58.961+0000] {subprocess.py:93} INFO - 	[SUCCESSFUL ] org.spark-project.spark#unused;1.0.0!unused.jar (283ms)
[2025-06-01T17:06:59.099+0000] {subprocess.py:93} INFO - downloading https://repo1.maven.org/maven2/org/apache/hadoop/hadoop-client-runtime/3.3.1/hadoop-client-runtime-3.3.1.jar ...
[2025-06-01T17:07:10.982+0000] {subprocess.py:93} INFO - 	[SUCCESSFUL ] org.apache.hadoop#hadoop-client-runtime;3.3.1!hadoop-client-runtime.jar (12021ms)
[2025-06-01T17:07:11.120+0000] {subprocess.py:93} INFO - downloading https://repo1.maven.org/maven2/org/lz4/lz4-java/1.7.1/lz4-java-1.7.1.jar ...
[2025-06-01T17:07:11.455+0000] {subprocess.py:93} INFO - 	[SUCCESSFUL ] org.lz4#lz4-java;1.7.1!lz4-java.jar (473ms)
[2025-06-01T17:07:11.591+0000] {subprocess.py:93} INFO - downloading https://repo1.maven.org/maven2/org/xerial/snappy/snappy-java/1.1.8.4/snappy-java-1.1.8.4.jar ...
[2025-06-01T17:07:12.335+0000] {subprocess.py:93} INFO - 	[SUCCESSFUL ] org.xerial.snappy#snappy-java;1.1.8.4!snappy-java.jar(bundle) (879ms)
[2025-06-01T17:07:12.470+0000] {subprocess.py:93} INFO - downloading https://repo1.maven.org/maven2/org/slf4j/slf4j-api/1.7.30/slf4j-api-1.7.30.jar ...
[2025-06-01T17:07:12.614+0000] {subprocess.py:93} INFO - 	[SUCCESSFUL ] org.slf4j#slf4j-api;1.7.30!slf4j-api.jar (278ms)
[2025-06-01T17:07:12.749+0000] {subprocess.py:93} INFO - downloading https://repo1.maven.org/maven2/org/apache/hadoop/hadoop-client-api/3.3.1/hadoop-client-api-3.3.1.jar ...
[2025-06-01T17:07:20.094+0000] {subprocess.py:93} INFO - 	[SUCCESSFUL ] org.apache.hadoop#hadoop-client-api;3.3.1!hadoop-client-api.jar (7480ms)
[2025-06-01T17:07:20.241+0000] {subprocess.py:93} INFO - downloading https://repo1.maven.org/maven2/org/apache/htrace/htrace-core4/4.1.0-incubating/htrace-core4-4.1.0-incubating.jar ...
[2025-06-01T17:07:20.905+0000] {subprocess.py:93} INFO - 	[SUCCESSFUL ] org.apache.htrace#htrace-core4;4.1.0-incubating!htrace-core4.jar (809ms)
[2025-06-01T17:07:21.042+0000] {subprocess.py:93} INFO - downloading https://repo1.maven.org/maven2/commons-logging/commons-logging/1.1.3/commons-logging-1.1.3.jar ...
[2025-06-01T17:07:21.195+0000] {subprocess.py:93} INFO - 	[SUCCESSFUL ] commons-logging#commons-logging;1.1.3!commons-logging.jar (290ms)
[2025-06-01T17:07:21.196+0000] {subprocess.py:93} INFO - :: resolution report :: resolve 47079ms :: artifacts dl 26247ms
[2025-06-01T17:07:21.197+0000] {subprocess.py:93} INFO - 	:: modules in use:
[2025-06-01T17:07:21.197+0000] {subprocess.py:93} INFO - 	com.google.code.findbugs#jsr305;3.0.0 from central in [default]
[2025-06-01T17:07:21.198+0000] {subprocess.py:93} INFO - 	commons-logging#commons-logging;1.1.3 from central in [default]
[2025-06-01T17:07:21.198+0000] {subprocess.py:93} INFO - 	org.apache.commons#commons-pool2;2.6.2 from central in [default]
[2025-06-01T17:07:21.199+0000] {subprocess.py:93} INFO - 	org.apache.hadoop#hadoop-client-api;3.3.1 from central in [default]
[2025-06-01T17:07:21.199+0000] {subprocess.py:93} INFO - 	org.apache.hadoop#hadoop-client-runtime;3.3.1 from central in [default]
[2025-06-01T17:07:21.200+0000] {subprocess.py:93} INFO - 	org.apache.htrace#htrace-core4;4.1.0-incubating from central in [default]
[2025-06-01T17:07:21.200+0000] {subprocess.py:93} INFO - 	org.apache.kafka#kafka-clients;2.8.0 from central in [default]
[2025-06-01T17:07:21.201+0000] {subprocess.py:93} INFO - 	org.apache.spark#spark-sql-kafka-0-10_2.12;3.2.0 from central in [default]
[2025-06-01T17:07:21.201+0000] {subprocess.py:93} INFO - 	org.apache.spark#spark-token-provider-kafka-0-10_2.12;3.2.0 from central in [default]
[2025-06-01T17:07:21.202+0000] {subprocess.py:93} INFO - 	org.lz4#lz4-java;1.7.1 from central in [default]
[2025-06-01T17:07:21.202+0000] {subprocess.py:93} INFO - 	org.slf4j#slf4j-api;1.7.30 from central in [default]
[2025-06-01T17:07:21.203+0000] {subprocess.py:93} INFO - 	org.spark-project.spark#unused;1.0.0 from central in [default]
[2025-06-01T17:07:21.203+0000] {subprocess.py:93} INFO - 	org.xerial.snappy#snappy-java;1.1.8.4 from central in [default]
[2025-06-01T17:07:21.204+0000] {subprocess.py:93} INFO - 	---------------------------------------------------------------------
[2025-06-01T17:07:21.204+0000] {subprocess.py:93} INFO - 	|                  |            modules            ||   artifacts   |
[2025-06-01T17:07:21.204+0000] {subprocess.py:93} INFO - 	|       conf       | number| search|dwnlded|evicted|| number|dwnlded|
[2025-06-01T17:07:21.205+0000] {subprocess.py:93} INFO - 	---------------------------------------------------------------------
[2025-06-01T17:07:21.206+0000] {subprocess.py:93} INFO - 	|      default     |   13  |   13  |   13  |   0   ||   13  |   13  |
[2025-06-01T17:07:21.206+0000] {subprocess.py:93} INFO - 	---------------------------------------------------------------------
[2025-06-01T17:07:21.207+0000] {subprocess.py:93} INFO - :: retrieving :: org.apache.spark#spark-submit-parent-8ae41390-c417-42fa-a2c7-9965c4dd3135
[2025-06-01T17:07:21.207+0000] {subprocess.py:93} INFO - 	confs: [default]
[2025-06-01T17:07:21.440+0000] {subprocess.py:93} INFO - 	13 artifacts copied, 0 already retrieved (59181kB/234ms)
[2025-06-01T17:07:21.804+0000] {subprocess.py:93} INFO - 25/06/01 17:07:21 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2025-06-01T17:07:22.409+0000] {subprocess.py:93} INFO - python3: can't open file '/app/log_consumer.py': [Errno 2] No such file or directory
[2025-06-01T17:07:22.430+0000] {subprocess.py:93} INFO - 25/06/01 17:07:22 INFO ShutdownHookManager: Shutdown hook called
[2025-06-01T17:07:22.432+0000] {subprocess.py:93} INFO - 25/06/01 17:07:22 INFO ShutdownHookManager: Deleting directory /tmp/spark-eacf549f-5efd-4164-b8a9-e26192bfb7e8
[2025-06-01T17:07:22.863+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2025-06-01T17:07:22.922+0000] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/bash.py", line 249, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2025-06-01T17:07:22.927+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=log_monitoring_dag, task_id=start_spark_consumer, run_id=scheduled__2025-06-01T16:50:00+00:00, execution_date=20250601T165000, start_date=20250601T170121, end_date=20250601T170722
[2025-06-01T17:07:22.955+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-01T17:07:22.956+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 47 for task start_spark_consumer (Bash command failed. The command returned a non-zero exit code 2.; 346)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3004, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3158, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3182, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/bash.py", line 249, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2025-06-01T17:07:22.984+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-06-01T17:07:23.003+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-01T17:07:23.009+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
